tests/integrations/test-integration.php

```diff
@@ -8,37 +8,36 @@
 namespace Automattic\VIP\Integrations;
 
 // phpcs:disable Squiz.Commenting.ClassComment.Missing, Squiz.Commenting.FunctionComment.Missing, Squiz.Commenting.FunctionComment.MissingParamComment
-// phpcs:disable WordPress.PHP.DiscouragedPHPFunctions.runtime_configuration_error_reporting
-// phpcs:disable WordPress.PHP.DevelopmentFunctions.error_log_set_error_handler
 
-use Automattic\VIP\Integrations\IntegrationVipConfig;
 use Env_Integration_Status;
 use ErrorException;
+use Org_Integration_Status;
 use PHPUnit\Framework\MockObject\MockObject;
 use WP_UnitTestCase;
 
+use function Automattic\Test\Utils\get_class_method_as_public;
+use function Automattic\Test\Utils\reset_custom_error_reporting;
+use function Automattic\Test\Utils\setup_custom_error_reporting;
+
 require_once __DIR__ . '/fake-integration.php';
+require_once __DIR__ . '/fake-multi-config-integration.php';
 
 class VIP_Integration_Test extends WP_UnitTestCase {
+	/**
+	 * Original error reporting.
+	 *
+	 * @var int
+	 */
 	private $original_error_reporting;
 
 	public function setUp(): void {
 		parent::setUp();
 
-		$this->original_error_reporting = error_reporting();
-		set_error_handler( static function ( int $errno, string $errstr ) {
-			if ( error_reporting() & $errno ) {
-				// phpcs:ignore WordPress.Security.EscapeOutput.ExceptionNotEscaped -- CLI
-				throw new ErrorException( $errstr, $errno ); // NOSONAR
-			}
-
-			return false;
-		}, E_USER_WARNING );
+		$this->original_error_reporting = setup_custom_error_reporting();
```

Comment:
Avoided duplication across files and added following methods:

- setup_custom_error_reporting
- reset_custom_error_reporting